tensorflow/lite/python/tflite_convert.py

```diff
@@ -156,6 +156,12 @@ def _convert_model(flags):
     converter.default_ranges_stats = (flags.default_ranges_min,
                                       flags.default_ranges_max)
 
+  # Set the max range of UINT8 as the default values of the default range
+  # variables.
+  if (flags.default_ranges_min is None) and (flags.default_ranges_max is None):
+      if converter.inference_type == lite_constants.QUANTIZED_UINT8:
+        converter.default_ranges_stats = (0, 128)
```

Comment:
This is not the intended usage of default_range_stats. default_range_stats is almost always the wrong thing to do. The converter currently expects the quantization info to be in the graph as FakeQuant nodes. If the info is missing you can use default_range_stats as a backup (for instance to do latency benchmarking without caring about accuracy.). There is no default value that is correct because the dynamic range of intermediate tensors varies depending on layer and model.